### PR TITLE
Add default repository labels

### PR DESCRIPTION
**Pull request by hillaryj**
#  #

## 🗣 Description ##

Adds a file to use as configuration for GitHub label standardization using https://github.com/cisagov/github-label-management

## 💭 Motivation and context ##

New repositories inherit labels from the `.github` repository, so using the tool and updating this `.github` repository will keep this repository and both new and old repos updated.

## 🧪 Testing ##

This was the file I used last year when standardizing repository labels, before I left 😭 . This will wrap up the work.

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] _All_ future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Comment by hillaryj** [comment]
Unfortunately, I can no longer add type-of-change labels, but this is otherwise all checked off.

**Review by jsf9k — APPROVED** [review]

**Comment by jsf9k** [comment]
The prodigal teammate returns!

File: repo-labels.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    "name": "upstream update"
+  },
+  {
+    "color": "#d4c5f9",
+    "description": "This issue or pull request involves a version bump",
+    "name": "version bump"
+  },
+  {
+    "color": "#ffffff",
+    "description": "This issue will not be worked on ",
```
Comment (dav3r) [inline repo-labels.json:84]: Remove an unintended space:
```suggestion
    "description": "This issue will not be worked on",
```

File: repo-labels.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    "name": "duplicate"
+  },
+  {
+    "color": "#b005bc",
+    "description": "This issue is an epic",
+    "name": "epic"
+  },
+  {
+    "color": "#0e8a16",
+    "description": "Good for newcomers",
```
Comment (dav3r) [inline repo-labels.json:39]: Update wording to align more closely with other descriptions:
```suggestion
    "description": "This issue is a good candidate for newcomers to work on",
```

File: repo-labels.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    "name": "improvement"
+  },
+  {
+    "color": "#fef2c0",
+    "description": "This doesn't seem right",
+    "name": "invalid"
+  },
+  {
+    "color": "#ce099a",
+    "description": "This pull request is ready to merge during the next Lineage kraken release",
```
Comment (dav3r) [inline repo-labels.json:54]: Make capitalization in description match that in the name field and add single quotes:
```suggestion
    "description": "This pull request is ready to merge during the next Lineage 'Kraken' release",
```

**Review by dav3r — CHANGES_REQUESTED** [review]
I noticed a few tiny things that I'd like to see changed, but great to hear from you @hillaryj 😄

**Comment by dav3r** [comment]
I'm closing this PR, as it's functionality has been superceded by https://github.com/cisagov/skeleton-generic/pull/118.

**Pull request closed without merging**